core/variant_parser.cpp

```diff
@@ -595,7 +595,7 @@ Error VariantParser::parse_value(Token &token, Variant &value, Stream *p_stream,
 			value = Quat(args[0], args[1], args[2], args[3]);
 			return OK;
 
-		} else if (id == "Rect3" || id == "AABB") {
```

Comment:
This was a pretty bad idea, renders existing scenes incompatible. Will add a fix asap.

Reply:
Ah, that's what is was for, sorry